Tests/MessagingPush/MessagingPushImplementationTest.swift

```diff
@@ -9,158 +9,20 @@ class MessagingPushImplementationTest: UnitTest {
     private var mockCustomerIO = CustomerIOInstanceMock()
     private var messagingPush: MessagingPushImplementation!
 
-    private let profileStoreMock = ProfileStoreMock()
     private let queueMock = QueueMock()
-    private let globalDataStoreMock = GlobalDataStoreMock()
-    private let deviceAttributesMock = DeviceAttributesProviderMock()
-    private let deviceInfoMock = DeviceInfoMock()
+    private let sdkInitializedUtilMock = SdkInitializedUtilMock()
 
     override func setUp() {
         super.setUp()
 
         mockCustomerIO.siteId = testSiteId
         messagingPush = MessagingPushImplementation(
             siteId: testSiteId,
-            profileStore: profileStoreMock,
-            backgroundQueue: queueMock,
-            globalDataStore: globalDataStoreMock,
             logger: log,
-            sdkConfig: sdkConfig,
             jsonAdapter: jsonAdapter,
-            deviceAttributesProvider: deviceAttributesMock,
-            dateUtil: dateUtilStub,
-            deviceInfo: deviceInfoMock
+            sdkConfig: sdkConfig,
+            backgroundQueue: queueMock,
+            sdkInitializedUtil: sdkInitializedUtilMock
         )
     }
-
-    // MARK: registerDeviceToken
```

Comment:
All of these tests have been moved to CustomerIOImplementation tests since that's where this code logic moved to.